cloud-azure/src/main/resources/templates/arm-v2-freeipa.ftl

```diff
@@ -160,6 +160,15 @@
                         </#list>
                      </#if>
                      },
+                   <#if instance.instance.availabilityZone?? && instance.instance.availabilityZone?has_content>
```

Comment:
where is this value coming from? where is the availabilityzone is set for an instance?

Reply:
This will be set as part of assigning availability zone to instance. This will be a subsequent change.